### PR TITLE
Min cannot be greater than Max

### PR DESCRIPTION
**Pull request by mrajashree**
https://github.com/rancher/rancher/issues/7446

Didn't add unit test because the ValidatePayload() of the mock driver only checks the values of min, max, action etc against the expectedConfig, and returns 500. So I'll add validation test after the PR: https://github.com/rancher/webhook-service/pull/26 is merged so the validation-test PRs won't clash 
@cjellick 
Also, it's okay to have min=max right?

### CODE REVIEW

**Comment by mrajashree** [comment]
Added test for this -> https://github.com/rancher/validation-tests/pull/272/files?diff=unified#diff-ea4a835e3cc399810b8f0225ad7130c5R107
@cjellick

File: drivers/scale_service.go [hunk 1]
```diff
@@ -44,6 +44,10 @@ func (s *ScaleServiceDriver) ValidatePayload(conf interface{}, apiClient client.
 		return http.StatusBadRequest, fmt.Errorf("Maximum scale not provided/invalid")
 	}
 
+	if config.Min > config.Max {
```
Comment (mrajashree) [inline drivers/scale_service.go]: @cjellick yes actually. I initially thought that there doesn't have to be an error thrown in that case. But I'll add that now to the condition

**Comment by mrajashree** [comment]
@cjellick tests for min >= max
https://github.com/rancher/validation-tests/pull/272/files#diff-ea4a835e3cc399810b8f0225ad7130c5R107

File: drivers/scale_service.go [hunk 2]
```diff
@@ -44,6 +44,10 @@ func (s *ScaleServiceDriver) ValidatePayload(conf interface{}, apiClient client.
 		return http.StatusBadRequest, fmt.Errorf("Maximum scale not provided/invalid")
 	}
 
+	if config.Min >= config.Max {
+		return http.StatusBadRequest, fmt.Errorf("Min cannot be greater than max")
```
Comment (cjellick) [inline drivers/scale_service.go]: Error statement is no longer accurate due to the >=. Change it to 'Max must be greater than min'.

**Comment by cjellick** [comment]
LGTM

**Pull request merged**